### PR TITLE
Update optical_flow.py

### PR DESCRIPTION
**Pull request by officialhemant511**
fix: #24424    Update DNN Optical Flow sample with RAFT model

### CODE REVIEW

File: samples/dnn/optical_flow.py [hunk 1]
```diff
@@ -59,17 +47,17 @@ def motion_to_color(self, flow):
...
 if __name__ == '__main__':
-    parser = argparse.ArgumentParser(description='Use this script to calculate optical flow using FlowNetv2',
+    parser = argparse.ArgumentParser(description='Use this script to calculate optical flow using RAFT',
                                      formatter_class=argparse.ArgumentDefaultsHelpFormatter)
     parser.add_argument('-input', '-i', required=True, help='Path to input video file. Skip this argument to capture frames from a camera.')
     parser.add_argument('--height', default=320, type=int, help='Input height')
     parser.add_argument('--width', default=448, type=int, help='Input width')
-    parser.add_argument('--proto', '-p', default='FlowNet2_deploy_anysize.prototxt', help='Path to prototxt.')
-    parser.add_argument('--model', '-m', default='FlowNet2_weights.caffemodel', help='Path to caffemodel.')
+    parser.add_argument('--proto', '-p', default='RAFT_deploy.prototxt', help='Path to prototxt.')
```
Comment (mshabunin) [inline samples/dnn/optical_flow.py:55]: Where can users download these files?

File: samples/dnn/optical_flow.py [hunk 2]
```diff
@@ -59,17 +47,17 @@ def motion_to_color(self, flow):
 
 
 if __name__ == '__main__':
-    parser = argparse.ArgumentParser(description='Use this script to calculate optical flow using FlowNetv2',
+    parser = argparse.ArgumentParser(description='Use this script to calculate optical flow using RAFT',
```
[the remark this answers is not in the document]
Comment (officialhemant511) [reply samples/dnn/optical_flow.py:50]: yeap.....i think it will work well...

**Comment by mshabunin** [comment]
Looks like this is garbage PR.

**Review by kayo09 — COMMENTED** [review]
What is going on, issue is closed but not merged?

**Pull request closed without merging**